app/src/main/java/org/mozilla/fenix/settings/search/AddSearchEngineFragment.kt

```diff
@@ -70,6 +71,7 @@ class AddSearchEngineFragment : Fragment(), CompoundButton.OnCheckedChangeListen
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
+        (activity as? HomeActivity)?.getSupportActionBarAndInflateIfNecessary()
```

Comment:
If we're doing a line-by-line replacement, should we put this in onResume when `show()` is called?

Then again, this is where "inflation" happens so maybe it's better to leave here. I'm not sure but I'm leaning towards the latter.